src/file_uploading.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, file, issue_key, user, auth_token, base_url):
         self.user = user
         self.auth_token = auth_token
         self.base_url = base_url
-        self.semaphore = BoundedSemaphore(2)
+        self.semaphore = BoundedSemaphore(8)
```

Comment:
SHould these be configurable?

Reply:
I don't see much value in making this configurable for now. Increasing this value further will not necessarily increase speed of upload, but will consume more memory. Reducing value will decrease upload speed.